src/AzureClient/EntryPoint/EntryPoint.cs

```diff
@@ -115,9 +115,62 @@ private ArgumentValue GetArgumentValue(System.Reflection.ParameterInfo parameter
             {
                 return new ArgumentValue.Result(Newtonsoft.Json.JsonConvert.DeserializeObject<Result>(parameterValue)!);
             }
+            else if (parameterType.IsQArray())
+            {
+                var arrayType = parameterType.GenericTypeArguments.FirstOrDefault() ??
+                    throw new ArgumentException($"Could not get the type of array.");
+                if (arrayType == typeof(bool)) {
```

Comment:
Style nit, but next-line brace style is preferred for C# code.
```suggestion
                if (arrayType == typeof(bool))
                {
```